bin/hdfcoinc/pycbc_coinc_findtrigs

```diff
@@ -126,8 +126,12 @@ class ReadByTemplate(object):
 
         if self.bank is not None:
             self.param = {}
-            for col in self.bank:
-                self.param[col] = self.bank[col][self.template_num]
+            if 'compressed_waveforms' in self.bank :
```

Comment:
Soumi, to be consistent, I don't this this should check for 'compressed_waveforms' only the existence of the parameters attribute. @cdcapano Do you agree?